sos/plugins/devicemapper.py

```diff
@@ -6,10 +6,15 @@
 #
 # See the LICENSE file in the source distribution for further information.
 
-from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin, \
+    ArchPlugin
 
 
-class DeviceMapper(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+class DeviceMapper(Plugin,
+                   RedHatPlugin,
+                   DebianPlugin,
+                   UbuntuPlugin,
+                   ArchPlugin):
```

Comment:
Maybe just personal stylistic preference, but listing plugins like https://github.com/sosreport/sos/blob/master/sos/plugins/kata_containers.py#L13-L14 seems better to me than scattere one plugin per one line. But as I wrote, this is rather a personal preference only.